notebooks/work-in-progress/fuel-price-imputation/fuel-price-imputation-zane.ipynb

```diff
@@ -0,0 +1,1133 @@
+{
```

Comment:
<p>I like this sanity check. Another useful output for model evaluation is to compute the residuals and relative % error and do some error analysis: plot histograms of error, do a <code>.describe()</code>, print the rows with the <code>.nlargest()</code> and <code>.nsmallest()</code> absolute and relative errors and make sure the inputs correspond with the outputs in a way that makes sense.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/1696/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Okay I'll look at doing some of this.</p>